ingestion/tests/integration/orm_profiler/test_orm_profiler_e2e.py

```diff
@@ -151,8 +154,10 @@ def tearDownClass(cls) -> None:
             hard_delete=True,
         )
 
-        NewUser.__table__.drop(bind=cls.engine)
         User.__table__.drop(bind=cls.engine)
+        NewUser.__table__.drop(bind=cls.engine)
+        cls.session.close()
+        return super().tearDownClass()
```

Comment:
we can remove this line, it won't do anything. The parent class does not have any implementation https://github.com/python/cpython/blob/702a5bc4637c82dc011e98b84f0cede98eb08dda/Lib/unittest/case.py#L479-L481